perf/perf_uprobe.py

```diff
@@ -44,8 +44,10 @@ def setUp(self):
         if 'Ubuntu' in self.distro_name:
             deps.extend(['linux-tools-common', 'linux-tools-%s' %
                          platform.uname()[2]])
-        elif self.distro_name in ['rhel', 'SuSE', 'fedora', 'centos']:
+        elif self.distro_name in ['SuSE', 'fedora', 'centos']:
             deps.extend(['perf'])
+        elif self.distro_name == 'rhel' and int(self.detected_distro.version) >= 9:
+            deps.extend(['perf', 'perf-debuginfo'])
```

Comment:
condition is ok on `RHEL9` case but I say move `perf` package in `rhel` distro case

Reply:
changed the script as suggested above. Please have a look at it once